packages/ckeditor5-table/src/tablecolumnresize/tablewidthresizecommand.js

```diff
@@ -51,10 +51,15 @@ export default class TableWidthResizeCommand extends TablePropertyCommand {
 		model.change( writer => {
 			if ( tableWidth ) {
 				writer.setAttribute( this.attributeName, tableWidth, table );
-				writer.setAttribute( 'columnWidths', columnWidths, table );
 			} else {
 				writer.removeAttribute( this.attributeName, table );
 			}
+
+			if ( columnWidths ) {
```

Comment:
Let's also document the fact that the model attribtue will be removed if respectable option param is not provided.

```
* @param {String} [options.tableWidth] The new table width. Model attribute gets removed if skipped.
```

And same for `options.columnWidths`.